aptos-move/aptos-release-builder/src/lib.rs

```diff
@@ -17,3 +22,33 @@ static CURRENT_RELEASE_CONFIG: Lazy<ReleaseConfig> =
 pub fn current_release_config() -> &'static ReleaseConfig {
     &CURRENT_RELEASE_CONFIG
 }
+
+static APTOS_CORE_PATH: OnceCell<PathBuf> = OnceCell::new();
+
+pub fn initialize_aptos_core_path(overriden_path: Option<PathBuf>) {
```

Comment:
This works, but it seems a little strange to me that we would use a global here instead of just passing the argument where we're using it since most of the code changed is our own code and we should be able to modify the function adding another arg

Reply:
I prefer a global because `--aptos-core-path` is a top-level arg of this tooling. It's widely used by all subcommand and it's supposed to be applied to all subcommands. Passing a thing you need everywhere just makes codes more verbose.